po/ja.po

```diff
@@ -0,0 +1,662 @@
+# French translation for LXD
+# Copyright (C) 2015 - LXD contributors
+# This file is distributed under the same license as LXD.
+# Hiroaki Nakamura <hnakamur@gmail.com>, 2015.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: LXD\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-03-12 13:58+1100\n"
+"PO-Revision-Date: 2015-02-26 02:05-0600\n"
+"Last-Translator: Stéphane Graber <stgraber@ubuntu.com\n"
+"Language-Team: French <fr@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
```

Comment:
The header should be updated for your translation. At least Last-Translator, Language-Team and possibly PO-Revision-Date (not very important for that one).